src/class-elements/private-getter-on-nested-class.case

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2019 Caio Lima (Igalia SL). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: PrivateName is available on inner classes (private getter)
+info: |
+  Updated Productions
+
+  CallExpression[Yield, Await]:
+    CoverCallExpressionAndAsyncArrowHead[?Yield, ?Await]
+    SuperCall[?Yield, ?Await]
+    CallExpression[?Yield, ?Await]Arguments[?Yield, ?Await]
+    CallExpression[?Yield, ?Await][Expression[+In, ?Yield, ?Await]]
+    CallExpression[?Yield, ?Await].IdentifierName
+    CallExpression[?Yield, ?Await]TemplateLiteral[?Yield, ?Await]
+    CallExpression[?Yield, ?Await].PrivateName
+
+template: default
+features: [class-methods-private]
+---*/
+
+//- elements
+get #m() { return 'test262'; }
+
+B = class {
+  method(o) {
+    return o.#m;
+  }
+}
+//- assertions
+let c = new C();
+let innerB = new c.B();
+assert.sameValue(innerB.method(c), 'test262');
```

Comment:
this test illustrates well a behavior I'm not really fond of. this is the sort of reference I'd avoid to have as a feature, but so far it is what it is.

At least, I'm glad it's now covered and I get a better idea on the feature.